repository/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -96,7 +96,5 @@
     <include file="db/changelog/db.changelog-rdm-9453.xml"/>
     <include file="db/changelog/db.changelog-rdm-9031.xml"/>
     <include file="db/changelog/db.changelog-rdm-9555.xml"/>
-    <include file="db/changelog/db.changelog-rdm-9994.xml"/>
-    <include file="db/changelog/db.changelog-rdm-9893.xml"/>
-    <include file="db/changelog/db.changelog-rdm-10238.xml"/>
```

Comment:
Will need to add these deleted ones back in since already in master

Reply:
https://github.com/hmcts/ccd-definition-store-api/pull/834